app/components/preprint-status-banner/component.js

```diff
@@ -161,6 +164,17 @@ export default Component.extend({
         return isBlank(this.get('reviewerComment'));
     }),
 
+    commentExceedsLimit: computed('reviewerComment', function() {
+        return this.get('reviewerComment.length') > COMMENT_LIMIT;
+    }),
+
+    commentLengthErrorMessage: computed(function () {
```

Comment:
This might be cleaner with the [`t` computed property macro](https://github.com/jamesarosen/ember-i18n/wiki/Doc:-Translating-Text#translation-computed-property-macro)